### DISCUSSION TITLE
Documentation for the SUV car?

### DISCUSSION THREAD

**Discussion by rhalaly** [microsoft/AirSim#4018] [Support Q&A]
Where can I find the documentation for the SUV car?

I need to know the max steering angle, the wheelbase length (the distance between the front and back wheels), and if it is possible, the function throttle->acceleration

**Comment by zimmy87** [comment]
Most of the documentation for car mode is located at https://microsoft.github.io/AirSim/using_car/

I think the information you're looking for is located in the source here or can at least be derived from the parameters passed to the UWheeledVehicleMovementComponent4W:

https://github.com/microsoft/AirSim/blob/master/Unreal/Plugins/AirSim/Source/Vehicles/Car/CarPawn.cpp#L86
